modules/polling/components/VotingWeight.tsx

```diff
@@ -73,9 +73,10 @@ export default function VotingWeight(): JSX.Element {
 
   const tooltipLabel = (
     <Box>
-      <Text as="p" sx={{ whiteSpace: 'normal' }}>
+      <Text as="p" sx={{ whiteSpace: 'normal', maxWidth: '400px' }}>
         {votingWeightCopy}
       </Text>
+      <br />
```

Comment:
We can add `mb: 3` (short for marginBottom) or however much space we need below this, to the `sx` prop on the <Text />` element, rather than a `<br /`> element below it. I think it's better to use the style system where we can.